Cargo.toml

```diff
@@ -28,7 +28,7 @@ members = [
 [dependencies]
 bitflags = "1"
 libc = "0.2"
-lmdb-sys = "0.8.0"
+lmdb-sys = { path = "lmdb-sys" }
```

Comment:
Sadly, crates.io doesn't support publishing a crate with a path dependency like this, and that doesn't seem likely to change, per https://github.com/rust-lang/cargo/issues/1565. So we'll have to publish the new version to crates.io (renaming it in the process to avoid the naming conflict with this version, which is controlled and updated more slowly by the upstream repo danburkert/lmdb-rs).

However, while I haven't tested it, a path dependency like this probably does work if you're depending on this crate via a Git repo reference rather than a crates.io reference. So this is probably fine for a branch where we track LMDB's upstream master branch but don't publish the crate on crates.io, and you depend on the Git repo directly.